SubscribeIT/ngDesk
# Inheritance mapping bug

Closes #216 

Test Case 1
Name:-Inheritance mapping while field creation
1. Login to ngdesk.
2. Goto -> modules- > tickets -> fields -> new.
3. Give valid name and select data type as relationship.
4. Fill all the fields.
5. Click on enable inheritance mapping.
6. Click on "+" icon atleast 5 times.
7. Observe the layout.
8. Observe no error.

Test Case 2
Name:-Inheritance mapping while field updating
1. Login to ngdesk.
2. Goto -> modules- > tickets -> fields -> click on any created relationship field.
3. Click on enable inheritance mapping.
4. Click on "+" icon atleast 5 times.
5. Observe the layout.
6. Observe no error.

Test Case 3
Name:-Items per page button alignment
1. Login to ngdesk.
2. Goto -> modules -> tickets -> fields.
3. Observe the items per page button.
4. Observe no error.

Screenshots :- 

![Screenshot from 2021-09-30 16-40-12](https://user-images.githubusercontent.com/89504119/135445964-2ecccb75-e40c-4f04-a502-0de61598b925.png)
![Screenshot from 2021-09-30 16-40-42](https://user-images.githubusercontent.com/89504119/135445966-e9fde08c-471a-4a2c-b72a-8c9d588d173f.png)
![Screenshot from 2021-10-01 16-34-46](https://user-images.githubusercontent.com/89504119/135609951-2e26382b-6f9b-44d0-8cdb-2e3910bf4c35.png)
